aws-ecr-replicationconfiguration/aws-ecr-replicationconfiguration.json

```diff
@@ -0,0 +1,125 @@
+{
+    "typeName": "AWS::ECR::ReplicationConfiguration",
+    "description": "The AWS::ECR::ReplicationConfiguration resource configures the replication destinations for an Amazon Elastic Container Registry (Amazon Private ECR). For more information, see https://docs.aws.amazon.com/AmazonECR/latest/userguide/replication.html",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-ecr.git",
+    "definitions": {
+        "Rules": {
+            "type": "array",
+            "minItems": 0,
```

Comment:
Does it make sense to allow replication configurations that have 0 rules?

Reply:
This is from the API - https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_ReplicationConfiguration.html